### ISSUE TITLE
limit 'insertHTML' command just to our editor

### ISSUE THREAD

**Issue by 4ndrea** [fraywing/textAngular#865]
The problem is that I want to give to the user possibility to add dynamic text only inside my editor and not all over the webpage. The code that I have right now inside and Angular controller is:
```
var editor = textAngularManager.retrieveEditor('myEditor').scope;
editor.wrapSelection('insertHTML', '<span>My Text</span>', true);
```

What am I doing wrong? Thanks !

**Comment by pbassut** [comment]
What you mean by "all over the page"? How does one will add html content to the page(unless the whole body has contenteditable in it)?
Could you clarify a bit more of what you need?

**Comment by 4ndrea** [comment]
If I click somewhere else on my webpage and then call the function that contains the code described, it inserts into that position on page outside my editor "My Text"

**Comment by pbassut** [comment]
Is this what you want: https://github.com/fraywing/textAngular/issues/817 ?

**Comment by 4ndrea** [comment]
Yes but no. I have this functionality. I insert code wherever I want, the problem is that I want to limit this only to my editor, only inside textAngular editor not all over the page :)

**Comment by pbassut** [comment]
So, could you provide a plunkr with the current behaviour and describe the intended behaviour?

**Comment by kawashimaken** [comment]
Try the following code, it might solve your problem.
notice the second line I added.
```
var editor = textAngularManager.retrieveEditor('myEditor').scope;
editor.displayElements.text.trigger('focus');
editor.wrapSelection('insertHTML', '<span>My Text</span>', true);
```
Good luck.
